pkg/webhooks/resource/generation/handler.go

```diff
@@ -300,6 +300,7 @@ func (h *generationHandler) processRequest(ctx context.Context, policyContext *e
 				}
 
 				ur := buildURSpec(kyvernov1beta1.Generate, pKey, rule.Name, generateutils.TriggerFromLabels(labels), deleteDownstream)
+				ur.Context = buildURContext(request, policyContext)
```

Comment:
This seems to break the generate test cases. Why do you think building the context is required here?

Reply:
I am working on this issue currently #4675, Here is my PR for it #8466. 
The idea is to use the resource UID as the label instead of the resource name. But to be able to retrieve the resource UID in `tryApplyResource()` the update request context would be needed. 
I have left a comment in #8466 for further discussion, please check it

Reply:
I understand the context of the PR now, please add this information in the PR description.

The failing test is this one: https://github.com/kyverno/kyverno/tree/main/test/conformance/kuttl/generate/clusterpolicy/standard/data/sync/cpol-data-sync-modify-downstream 

It's not easy to tell why without checking the kyverno logs more closely while the test is running.

Reply:
After creating a kind cluster, I ran `make-debug-deploy` and then ran the Kyverno Admission and Background Controller in debug mode

Then I ran these from this directory (https://github.com/kyverno/kyverno/tree/main/test/conformance/kuttl/generate/clusterpolicy/standard/data/sync/cpol-data-sync-modify-downstream)

1. `kubectl apply -f 01-manifests.yaml `
2. `kubectl apply -f 02-ns.yaml `
3. `kubectl apply -f 03-modify.yaml `

Here's the logs from the Kyverno Admission Controller in `level 4`:
```
I0929 16:37:23.577087   39172 version.go:49] setup/version "msg"="version" "hash"="c023f972ba1708fca2896ef4f14bbad04fe3cb07" "time"="2023-09-29T10:52:47Z" "version"="(devel)"
I0929 16:37:23.577153   39172 flag.go:240] setup/flag "msg"="" "add_dir_header"=false
I0929 16:37:23.577175   39172 flag.go:240] setup/flag "msg"="" "admissionReports"=true
I0929 16:37:23.577193   39172 flag.go:240] setup/flag "msg"="" "allowInsecureRegistry"=false
I0929 16:37:23.577210   39172 flag.go:240] setup/flag "msg"="" "alsologtostderr"=false
I0929 16:37:23.577229   39172 flag.go:240] setup/flag "msg"="" "autoUpdateWebhooks"=true
I0929 16:37:23.577260   39172 flag.go:240] setup/flag "msg"="" "backgroundServiceAccountName"="system:serviceaccount:kyverno:kyverno-background-controller"
I0929 16:37:23.577278   39172 flag.go:240] setup/flag "msg"="" "caSecretName"="kyverno-svc.kyverno.svc.kyverno-tls-ca"
I0929 16:37:23.577307   39172 flag.go:240] setup/flag "msg"="" "clientRateLimitBurst"=50
I0929 16:37:23.577368   39172 flag.go:240] setup/flag "msg"="" "clientRateLimitQPS"=20
I0929 16:37:23.577398   39172 flag.go:240] setup/flag "msg"="" "disableMetrics"=false
I0929 16:37:23.577422   39172 flag.go:240] setup/flag "msg"="" "dumpPayload"=false
I0929 16:37:23.577446   39172 flag.go:240] setup/flag "msg"="" "enableConfigMapCaching"=true
I0929 16:37:23.577494   39172 flag.go:240] setup/flag "msg"="" "enableDeferredLoading"=
I0929 16:37:23.577533   39172 flag.go:240] setup/flag "msg"="" "enablePolicyException"=true
I0929 16:37:23.577564   39172 flag.go:240] setup/flag "msg"="" "enableTracing"=false
I0929 16:37:23.577599   39172 flag.go:240] setup/flag "msg"="" "enableTuf"=false
I0929 16:37:23.577632   39172 flag.go:240] setup/flag "msg"="" "exceptionNamespace"=""
I0929 16:37:23.577679   39172 flag.go:240] setup/flag "msg"="" "forceFailurePolicyIgnore"=
I0929 16:37:23.577716   39172 flag.go:240] setup/flag "msg"="" "generateValidatingAdmissionPolicy"=false
I0929 16:37:23.577749   39172 flag.go:240] setup/flag "msg"="" "imagePullSecrets"=""
I0929 16:37:23.577787   39172 flag.go:240] setup/flag "msg"="" "imageSignatureRepository"=""
I0929 16:37:23.577822   39172 flag.go:240] setup/flag "msg"="" "imageVerifyCacheEnabled"=true
I0929 16:37:23.577863   39172 flag.go:240] setup/flag "msg"="" "imageVerifyCacheMaxSize"=1000
I0929 16:37:23.577906   39172 flag.go:240] setup/flag "msg"="" "imageVerifyCacheTTLDuration"=3600000000000
I0929 16:37:23.577940   39172 flag.go:240] setup/flag "msg"="" "kubeconfig"="/home/ubuntu/.kube/config"
I0929 16:37:23.577974   39172 flag.go:240] setup/flag "msg"="" "leaderElectionRetryPeriod"=2000000000
I0929 16:37:23.578035   39172 flag.go:240] setup/flag "msg"="" "log_backtrace_at"={}
I0929 16:37:23.578070   39172 flag.go:240] setup/flag "msg"="" "log_dir"=""
I0929 16:37:23.578109   39172 flag.go:240] setup/flag "msg"="" "log_file"=""
I0929 16:37:23.578135   39172 flag.go:240] setup/flag "msg"="" "log_file_max_size"=1800
I0929 16:37:23.578152   39172 flag.go:240] setup/flag "msg"="" "loggingFormat"="text"
I0929 16:37:23.578170   39172 flag.go:240] setup/flag "msg"="" "logtostderr"=true
I0929 16:37:23.578205   39172 flag.go:240] setup/flag "msg"="" "maxQueuedEvents"=1000
I0929 16:37:23.578245   39172 flag.go:240] setup/flag "msg"="" "metricsPort"="8000"
I0929 16:37:23.578280   39172 flag.go:240] setup/flag "msg"="" "omit-events"=""
I0929 16:37:23.578313   39172 flag.go:240] setup/flag "msg"="" "one_output"=false
I0929 16:37:23.578395   39172 flag.go:240] setup/flag "msg"="" "otelCollector"="opentelemetrycollector.kyverno.svc.cluster.local"
I0929 16:37:23.578432   39172 flag.go:240] setup/flag "msg"="" "otelConfig"="prometheus"
I0929 16:37:23.578465   39172 flag.go:240] setup/flag "msg"="" "profile"=false
I0929 16:37:23.578504   39172 flag.go:240] setup/flag "msg"="" "profileAddress"=""
I0929 16:37:23.578535   39172 flag.go:240] setup/flag "msg"="" "profilePort"="6060"
I0929 16:37:23.578566   39172 flag.go:240] setup/flag "msg"="" "protectManagedResources"=
I0929 16:37:23.578593   39172 flag.go:240] setup/flag "msg"="" "registryCredentialHelpers"=""
I0929 16:37:23.578621   39172 flag.go:240] setup/flag "msg"="" "serverIP"="172.31.0.165:9443"
I0929 16:37:23.578650   39172 flag.go:240] setup/flag "msg"="" "servicePort"=443
I0929 16:37:23.578677   39172 flag.go:240] setup/flag "msg"="" "skip_headers"=false
I0929 16:37:23.578707   39172 flag.go:240] setup/flag "msg"="" "skip_log_headers"=false
I0929 16:37:23.578783   39172 flag.go:240] setup/flag "msg"="" "stderrthreshold"={"Severity":2}
I0929 16:37:23.578818   39172 flag.go:240] setup/flag "msg"="" "tlsSecretName"="kyverno-svc.kyverno.svc.kyverno-tls-pair"
I0929 16:37:23.578849   39172 flag.go:240] setup/flag "msg"="" "tracingAddress"=""
I0929 16:37:23.578872   39172 flag.go:240] setup/flag "msg"="" "tracingCreds"=""
I0929 16:37:23.578902   39172 flag.go:240] setup/flag "msg"="" "tracingPort"="4317"
I0929 16:37:23.578931   39172 flag.go:240] setup/flag "msg"="" "transportCreds"=""
I0929 16:37:23.578966   39172 flag.go:240] setup/flag "msg"="" "tufMirror"="https://tuf-repo-cdn.sigstore.dev"
I0929 16:37:23.579000   39172 flag.go:240] setup/flag "msg"="" "tufRoot"=""
I0929 16:37:23.579053   39172 flag.go:240] setup/flag "msg"="" "v"=4
I0929 16:37:23.579096   39172 flag.go:240] setup/flag "msg"="" "vmodule"={}
I0929 16:37:23.579118   39172 flag.go:240] setup/flag "msg"="" "webhookRegistrationTimeout"=120000000000
I0929 16:37:23.579153   39172 flag.go:240] setup/flag "msg"="" "webhookTimeout"=10
I0929 16:37:23.579186   39172 maxprocs.go:12] setup/maxprocs "msg"="setup maxprocs..." 
I0929 16:37:23.579696   39172 maxprocs.go:16] setup/maxprocs "msg"="maxprocs: Leaving GOMAXPROCS=4: CPU quota undefined" 
I0929 16:37:23.579731   39172 signal.go:16] setup/signals "msg"="setup signals..." 
I0929 16:37:23.580249   39172 client.go:41] setup/kube-client "msg"="create kube client..." "burst"=50 "kubeconfig"="/home/ubuntu/.kube/config" "qps"=20
I0929 16:37:23.583615   39172 reflector.go:289] Starting reflector *v1.ConfigMap (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.583666   39172 reflector.go:325] Listing and watching *v1.ConfigMap from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.684428   39172 shared_informer.go:341] caches populated
I0929 16:37:23.684510   39172 metricsconfig.go:116] config "msg"="metricsRefreshInterval not set" "name"="kyverno-metrics" "namespace"="kyverno"
I0929 16:37:23.684601   39172 metricsconfig.go:138] config "msg"="namespaces configured" "name"="kyverno-metrics" "namespace"="kyverno" "namespaces"="{\"exclude\":[],\"include\":[]}"
I0929 16:37:23.684777   39172 metrics.go:18] setup/metrics "msg"="setup metrics..." "collector"="opentelemetrycollector.kyverno.svc.cluster.local" "creds"="" "otel"="prometheus" "port"="8000"
I0929 16:37:23.685003   39172 run.go:58] metrics-config-controller "msg"="starting ..." 
I0929 16:37:23.685080   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:23.685166   39172 reflector.go:289] Starting reflector *v1.ConfigMap (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.685378   39172 run.go:71] metrics-config-controller/worker "msg"="starting worker" "id"=0
E0929 16:37:23.685409   39172 metrics.go:54] setup/metrics "msg"="failed to enable metrics" "error"="listen tcp :8000: bind: address already in use" "address"=":8000"
I0929 16:37:23.685390   39172 reflector.go:325] Listing and watching *v1.ConfigMap from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.785535   39172 shared_informer.go:341] caches populated
I0929 16:37:23.786559   39172 config.go:338] config "msg"="filters configured" "filters"=[{"Group":"*","Version":"*","Kind":"*","Subresource":"*","Namespace":"kyverno","Name":"*"},{"Group":"*","Version":"*","Kind":"Event","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"*","Subresource":"*","Namespace":"kube-system","Name":"*"},{"Group":"*","Version":"*","Kind":"*","Subresource":"*","Namespace":"kube-public","Name":"*"},{"Group":"*","Version":"*","Kind":"*","Subresource":"*","Namespace":"kube-node-lease","Name":"*"},{"Group":"*","Version":"*","Kind":"Node","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"Node","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"APIService","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"APIService","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"TokenReview","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"SubjectAccessReview","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"SelfSubjectAccessReview","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"Binding","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"binding","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ReplicaSet","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ReplicaSet","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"AdmissionReport","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"AdmissionReport","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterAdmissionReport","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterAdmissionReport","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"BackgroundScanReport","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"BackgroundScanReport","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterBackgroundScanReport","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterBackgroundScanReport","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:admission-controller"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:admission-controller:core"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:admission-controller:additional"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:background-controller"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:background-controller:core"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:background-controller:additional"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:cleanup-controller"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:cleanup-controller:core"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:cleanup-controller:additional"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:reports-controller"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:reports-controller:core"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:reports-controller:additional"},{"Group":"*","Version":"*","Kind":"ClusterRoleBinding","Subresource":"","Namespace":"*","Name":"kyverno:admission-controller"},{"Group":"*","Version":"*","Kind":"ClusterRoleBinding","Subresource":"","Namespace":"*","Name":"kyverno:background-controller"},{"Group":"*","Version":"*","Kind":"ClusterRoleBinding","Subresource":"","Namespace":"*","Name":"kyverno:cleanup-controller"},{"Group":"*","Version":"*","Kind":"ClusterRoleBinding","Subresource":"","Namespace":"*","Name":"kyverno:reports-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Role","Subresource":"","Namespace":"kyverno","Name":"kyverno:admission-controller"},{"Group":"*","Version":"*","Kind":"Role","Subresource":"","Namespace":"kyverno","Name":"kyverno:background-controller"},{"Group":"*","Version":"*","Kind":"Role","Subresource":"","Namespace":"kyverno","Name":"kyverno:cleanup-controller"},{"Group":"*","Version":"*","Kind":"Role","Subresource":"","Namespace":"kyverno","Name":"kyverno:reports-controller"},{"Group":"*","Version":"*","Kind":"RoleBinding","Subresource":"","Namespace":"kyverno","Name":"kyverno:admission-controller"},{"Group":"*","Version":"*","Kind":"RoleBinding","Subresource":"","Namespace":"kyverno","Name":"kyverno:background-controller"},{"Group":"*","Version":"*","Kind":"RoleBinding","Subresource":"","Namespace":"kyverno","Name":"kyverno:cleanup-controller"},{"Group":"*","Version":"*","Kind":"RoleBinding","Subresource":"","Namespace":"kyverno","Name":"kyverno:reports-controller"},{"Group":"*","Version":"*","Kind":"ConfigMap","Subresource":"","Namespace":"kyverno","Name":"kyverno"},{"Group":"*","Version":"*","Kind":"ConfigMap","Subresource":"","Namespace":"kyverno","Name":"kyverno-metrics"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller-*"},{"Group":"*","Version":"*","Kind":"Job","Subresource":"","Namespace":"kyverno","Name":"kyverno-hook-pre-delete"},{"Group":"*","Version":"*","Kind":"Job","Subresource":"*","Namespace":"kyverno","Name":"kyverno-hook-pre-delete"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-svc"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-svc"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-svc-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-svc-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller-metrics"},{"Group":"*","Version":"*","Kind":"ServiceMonitor","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"ServiceMonitor","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"ServiceMonitor","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"ServiceMonitor","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Secret","Subresource":"","Namespace":"kyverno","Name":"kyverno-svc.kyverno.svc.*"},{"Group":"*","Version":"*","Kind":"Secret","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller.kyverno.svc.*"}] "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.786701   39172 config.go:347] config "msg"="defaultRegistry configured" "defaultRegistry"="docker.io" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.786737   39172 config.go:363] config "msg"="enableDefaultRegistryMutation configured" "enableDefaultRegistryMutation"="true" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.786775   39172 config.go:372] config "msg"="excludedGroups configured" "excludeGroups"=["system:nodes"] "includeGroups"=null "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.786803   39172 config.go:377] config "msg"="excludeUsernames not set" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.786831   39172 config.go:385] config "msg"="excludeRoles not set" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.786859   39172 config.go:393] config "msg"="excludeClusterRoles not set" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.786892   39172 config.go:409] config "msg"="generateSuccessEvents configured" "generateSuccessEvents"="false" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.787033   39172 config.go:423] config "msg"="webhooks configured" "name"="kyverno" "namespace"="kyverno" "webhooks"="[{\"namespaceSelector\": {\"matchExpressions\": [{\"key\":\"kubernetes.io/metadata.name\",\"operator\":\"NotIn\",\"values\":[\"kyverno\"]}]}}]"
I0929 16:37:23.787071   39172 config.go:429] config "msg"="webhookAnnotations not set" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.787102   39172 config.go:443] config "msg"="matchConditions not set" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.787155   39172 registry.go:18] setup/registry-client "msg"="setup registry client..." "insecure"=false "secrets"=""
I0929 16:37:23.787264   39172 run.go:58] config-controller "msg"="starting ..." 
I0929 16:37:23.787289   39172 reflector.go:289] Starting reflector *v1.Secret (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.787335   39172 reflector.go:325] Listing and watching *v1.Secret from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.787313   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:23.787668   39172 run.go:71] config-controller/worker "msg"="starting worker" "id"=0
I0929 16:37:23.887553   39172 shared_informer.go:341] caches populated
I0929 16:37:23.887665   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:23.887751   39172 imageverifycache.go:12] setup/image-verify-cache "msg"="setup image verify cache..." "enabled"=true "maxsize"=1000 "ttl"=3600000000000
I0929 16:37:23.888033   39172 client.go:41] setup/kube-client "msg"="create kube client..." "burst"=50 "kubeconfig"="/home/ubuntu/.kube/config" "qps"=20
I0929 16:37:23.890024   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:23.890434   39172 client.go:49] setup/kyverno-client "msg"="create kyverno client..." "burst"=50 "kubeconfig"="/home/ubuntu/.kube/config" "qps"=20
I0929 16:37:23.892180   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:23.892307   39172 client.go:57] setup/dynamic-client "msg"="create dynamic client..." "burst"=50 "kubeconfig"="/home/ubuntu/.kube/config" "qps"=20
I0929 16:37:23.894104   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:23.894172   39172 client.go:73] setup/apiserver-client "msg"="create apiserver client..." "burst"=50 "kubeconfig"="/home/ubuntu/.kube/config" "qps"=20
I0929 16:37:23.895911   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:23.896005   39172 client.go:81] setup/d-client "msg"="create the kyverno dynamic client..." "burst"=50 "kubeconfig"="/home/ubuntu/.kube/config" "qps"=20
I0929 16:37:23.896179   39172 reflector.go:289] Starting reflector *v1.Secret (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.896222   39172 reflector.go:325] Listing and watching *v1.Secret from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.896210   39172 reflector.go:289] Starting reflector *v1.Secret (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.896299   39172 reflector.go:325] Listing and watching *v1.Secret from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.996948   39172 shared_informer.go:341] caches populated
I0929 16:37:24.481058   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.481429   39172 engine.go:87] setup/configmap-resolver "msg"="setup config map resolver..." "enableConfigMapCaching"=true
I0929 16:37:24.481829   39172 reflector.go:289] Starting reflector *v1.ConfigMap (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.481869   39172 reflector.go:325] Listing and watching *v1.ConfigMap from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.582492   39172 shared_informer.go:341] caches populated
I0929 16:37:24.582579   39172 engine.go:62] setup/exception-selector "msg"="setup exception selector..." "enablePolicyException"=true "exceptionNamespace"=""
I0929 16:37:24.582769   39172 reflector.go:289] Starting reflector *v2alpha1.PolicyException (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.582810   39172 reflector.go:325] Listing and watching *v2alpha1.PolicyException from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.683008   39172 shared_informer.go:341] caches populated
I0929 16:37:24.683071   39172 engine.go:41] setup/engine "msg"="setup engine..." 
I0929 16:37:24.683114   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.683364   39172 reflector.go:289] Starting reflector *v1.Policy (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.683405   39172 reflector.go:325] Listing and watching *v1.Policy from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.683396   39172 reflector.go:289] Starting reflector *v1.Deployment (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.683476   39172 reflector.go:325] Listing and watching *v1.Deployment from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.683494   39172 reflector.go:289] Starting reflector *v1.ClusterPolicy (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.683642   39172 reflector.go:325] Listing and watching *v1.ClusterPolicy from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.783704   39172 shared_informer.go:341] caches populated
I0929 16:37:24.783738   39172 shared_informer.go:341] caches populated
I0929 16:37:24.783763   39172 shared_informer.go:341] caches populated
I0929 16:37:24.783804   39172 controller.go:66] policycache-controller "msg"="warming up ..." 
I0929 16:37:24.783837   39172 controller.go:91] policycache-controller "msg"="warm up done" 
I0929 16:37:24.784006   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.784032   39172 controller.go:32] setup/controllers "msg"="starting controller" "name"="policycache-controller" "workers"=3
I0929 16:37:24.784092   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.784011   39172 controller.go:152] EventGenerator "msg"="start" 
I0929 16:37:24.784101   39172 run.go:58] policycache-controller "msg"="starting ..." 
I0929 16:37:24.784207   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.784061   39172 leaderelection.go:250] attempting to acquire leader lease kyverno/kyverno...
I0929 16:37:24.784316   39172 run.go:71] policycache-controller/worker "msg"="starting worker" "id"=2
I0929 16:37:24.784371   39172 run.go:71] policycache-controller/worker "msg"="starting worker" "id"=0
I0929 16:37:24.784424   39172 run.go:71] policycache-controller/worker "msg"="starting worker" "id"=1
I0929 16:37:24.784159   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:24.784624   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.784671   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.784732   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:24.784789   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.784841   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.784883   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:24.784932   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.784985   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.785041   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:24.785103   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.785158   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.785214   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:24.785279   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.785322   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.785427   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:24.785489   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.785541   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.785597   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:24.785655   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.785707   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.785763   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:24.785811   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.785857   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.785897   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:24.785953   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.785992   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:24.786115   39172 reflector.go:289] Starting reflector *v1beta1.UpdateRequest (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.786154   39172 reflector.go:325] Listing and watching *v1beta1.UpdateRequest from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.786327   39172 reflector.go:289] Starting reflector *v1.RoleBinding (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.786677   39172 reflector.go:325] Listing and watching *v1.RoleBinding from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.786430   39172 reflector.go:289] Starting reflector *v1.ClusterRoleBinding (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.787211   39172 reflector.go:325] Listing and watching *v1.ClusterRoleBinding from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.786539   39172 reflector.go:289] Starting reflector *v1.Namespace (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.787667   39172 reflector.go:325] Listing and watching *v1.Namespace from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.791446   39172 leaderelection.go:260] successfully acquired lease kyverno/kyverno
I0929 16:37:24.791639   39172 leaderelection.go:97] setup/leader-election "msg"="still leading" "id"="dummy"
I0929 16:37:24.791669   39172 leaderelection.go:83] setup/leader-election "msg"="started leading" "id"="dummy"
I0929 16:37:24.791986   39172 reflector.go:289] Starting reflector *v1.MutatingWebhookConfiguration (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.792034   39172 reflector.go:325] Listing and watching *v1.MutatingWebhookConfiguration from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.792070   39172 reflector.go:289] Starting reflector *v1.Policy (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.792104   39172 reflector.go:325] Listing and watching *v1.Policy from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.792266   39172 reflector.go:289] Starting reflector *v1.ValidatingWebhookConfiguration (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.792300   39172 reflector.go:325] Listing and watching *v1.ValidatingWebhookConfiguration from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.792387   39172 reflector.go:289] Starting reflector *v1.ClusterRole (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.792423   39172 reflector.go:325] Listing and watching *v1.ClusterRole from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.791987   39172 reflector.go:289] Starting reflector *v1.ClusterPolicy (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.792495   39172 reflector.go:325] Listing and watching *v1.ClusterPolicy from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.792668   39172 reflector.go:289] Starting reflector *v1.Lease (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.792706   39172 reflector.go:325] Listing and watching *v1.Lease from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.886474   39172 shared_informer.go:341] caches populated
I0929 16:37:24.886508   39172 shared_informer.go:341] caches populated
I0929 16:37:24.886525   39172 shared_informer.go:341] caches populated
I0929 16:37:24.886547   39172 shared_informer.go:341] caches populated
I0929 16:37:24.886563   39172 shared_informer.go:341] caches populated
I0929 16:37:24.886581   39172 shared_informer.go:341] caches populated
I0929 16:37:24.886600   39172 shared_informer.go:341] caches populated
I0929 16:37:24.886616   39172 shared_informer.go:341] caches populated
I0929 16:37:24.886667   39172 server.go:209] webhooks "msg"="starting service" 
I0929 16:37:24.886733   39172 server.go:204] webhooks "msg"="started serving requests" "addr"=":9443"
I0929 16:37:24.892889   39172 shared_informer.go:341] caches populated
I0929 16:37:24.892909   39172 shared_informer.go:341] caches populated
I0929 16:37:24.892929   39172 shared_informer.go:341] caches populated
I0929 16:37:24.892948   39172 shared_informer.go:341] caches populated
I0929 16:37:24.892955   39172 shared_informer.go:341] caches populated
I0929 16:37:24.892962   39172 shared_informer.go:341] caches populated
I0929 16:37:24.892979   39172 shared_informer.go:341] caches populated
I0929 16:37:24.893059   39172 controller.go:32] setup/leader/controllers "msg"="starting controller" "name"="exception-webhook-controller" "workers"=1
I0929 16:37:24.893099   39172 run.go:58] exception-webhook-controller "msg"="starting ..." 
I0929 16:37:24.893110   39172 controller.go:32] setup/leader/controllers "msg"="starting controller" "name"="webhook-controller" "workers"=2
I0929 16:37:24.893139   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.893163   39172 run.go:58] webhook-controller "msg"="starting ..." 
I0929 16:37:24.893191   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.893240   39172 run.go:71] exception-webhook-controller/worker "msg"="starting worker" "id"=0
I0929 16:37:24.893285   39172 run.go:80] webhook-controller/routine "msg"="starting routine" "id"=0
I0929 16:37:24.893361   39172 run.go:112] exception-webhook-controller/worker "msg"="Dropping request from the queue" "error"="secret \"kyverno-svc.kyverno.svc.kyverno-tls-ca\" not found" "id"=0 "obj"="kyverno-exception-validating-webhook-cfg"
I0929 16:37:24.893363   39172 run.go:71] webhook-controller/worker "msg"="starting worker" "id"=0
I0929 16:37:24.893395   39172 run.go:71] webhook-controller/worker "msg"="starting worker" "id"=1
I0929 16:37:24.893454   39172 run.go:112] webhook-controller/worker "msg"="Dropping request from the queue" "error"="secret \"kyverno-svc.kyverno.svc.kyverno-tls-ca\" not found" "id"=0 "obj"="kyverno-policy-validating-webhook-cfg"
I0929 16:37:24.893479   39172 run.go:112] webhook-controller/worker "msg"="Dropping request from the queue" "error"="secret \"kyverno-svc.kyverno.svc.kyverno-tls-ca\" not found" "id"=1 "obj"="kyverno-policy-mutating-webhook-cfg"
I0929 16:37:24.893518   39172 run.go:112] webhook-controller/worker "msg"="Dropping request from the queue" "error"="secret \"kyverno-svc.kyverno.svc.kyverno-tls-ca\" not found" "id"=0 "obj"="kyverno-resource-mutating-webhook-cfg"
I0929 16:37:24.893535   39172 run.go:112] webhook-controller/worker "msg"="Dropping request from the queue" "error"="secret \"kyverno-svc.kyverno.svc.kyverno-tls-ca\" not found" "id"=1 "obj"="kyverno-resource-validating-webhook-cfg"
I0929 16:37:24.893072   39172 controller.go:32] setup/leader/controllers "msg"="starting controller" "name"="certmanager-controller" "workers"=1
I0929 16:37:24.893579   39172 run.go:112] webhook-controller/worker "msg"="Dropping request from the queue" "error"="secret \"kyverno-svc.kyverno.svc.kyverno-tls-ca\" not found" "id"=0 "obj"="kyverno-verify-mutating-webhook-cfg"
I0929 16:37:24.893610   39172 run.go:58] certmanager-controller "msg"="starting ..." 
I0929 16:37:24.893652   39172 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.893774   39172 run.go:71] certmanager-controller/worker "msg"="starting worker" "id"=0
I0929 16:37:24.893788   39172 run.go:80] certmanager-controller/routine "msg"="starting routine" "id"=0
I0929 16:37:26.050660   39172 retry.go:23] certmanager-controller "msg"="failed to renew CA" "reason"="failed to validate certs (secrets \"kyverno-svc.kyverno.svc.kyverno-tls-pair\" not found)"
E0929 16:37:26.090028   39172 controller.go:258] webhook-controller "msg"="failed to get lease" "error"="lease.coordination.k8s.io \"kyverno-health\" not found" 
E0929 16:37:26.124827   39172 controller.go:258] webhook-controller "msg"="failed to get lease" "error"="lease.coordination.k8s.io \"kyverno-health\" not found" 
E0929 16:37:26.151614   39172 controller.go:258] webhook-controller "msg"="failed to get lease" "error"="lease.coordination.k8s.io \"kyverno-health\" not found" 
E0929 16:37:26.154885   39172 controller.go:258] webhook-controller "msg"="failed to get lease" "error"="lease.coordination.k8s.io \"kyverno-health\" not found" 
I0929 16:38:40.061823   39172 vars.go:375]  "msg"="variable substituted" "path"="/generate/namespace" "value"=null "variable"="{{request.object.metadata.name}}"
I0929 16:38:40.089169   39172 controller.go:45] setup/cluster-policy "msg"="resource added" "name"="cpol-data-sync-modify-downstream-policy" "type"="ClusterPolicy"
I0929 16:38:40.089362   39172 store.go:41] policycache "msg"="policy is added to cache" "key"="cpol-data-sync-modify-downstream-policy"
I0929 16:38:40.103644   39172 store.go:41] policycache "msg"="policy is added to cache" "key"="cpol-data-sync-modify-downstream-policy"
I0929 16:39:11.600850   39172 handlers.go:101] webhooks/resource/validate "msg"="received an admission request in validating webhook" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "gvk"={"group":"","version":"v1","kind":"Namespace"} "gvr"={"group":"","version":"v1","resource":"namespaces"} "kind"="Namespace" "name"="trainer" "namespace"="trainer" "operation"="CREATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"Namespace"} "roles"=null "uid"="1a91a494-b510-4e0c-b2af-5cd337c54bf4" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}
I0929 16:39:11.600957   39172 handlers.go:115] webhooks/resource/validate "msg"="processing policies for validate admission request" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "generate"=1 "gvk"={"group":"","version":"v1","kind":"Namespace"} "gvr"={"group":"","version":"v1","resource":"namespaces"} "kind"="Namespace" "mutate"=0 "name"="trainer" "namespace"="trainer" "operation"="CREATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"Namespace"} "roles"=null "uid"="1a91a494-b510-4e0c-b2af-5cd337c54bf4" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]} "validate"=0
I0929 16:39:11.603274   39172 context.go:255] context "msg"="Adding service account" "service account name"="" "service account namespace"=""
I0929 16:39:11.603430   39172 block.go:33] webhooks/resource/validate "msg"="allowing admission request" "action"="validate" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "gvk"={"group":"","version":"v1","kind":"Namespace"} "gvr"={"group":"","version":"v1","resource":"namespaces"} "kind"="Namespace" "name"="trainer" "namespace"="trainer" "operation"="CREATE" "resource"="trainer/Namespace/trainer" "resource.gvk"={"Group":"","Version":"v1","Kind":"Namespace"} "roles"=null "uid"="1a91a494-b510-4e0c-b2af-5cd337c54bf4" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}
I0929 16:39:11.603466   39172 controller.go:121] EventGenerator "msg"="generating events" "count"=0
I0929 16:39:14.858740   39172 handler.go:109] webhooks/resource/validate "msg"="handle trigger resource operation for generate" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "gvk"={"group":"","version":"v1","kind":"Namespace"} "gvr"={"group":"","version":"v1","resource":"namespaces"} "kind"="Namespace" "name"="trainer" "namespace"="trainer" "operation"="CREATE" "policies"=1 "resource.gvk"={"Group":"","Version":"v1","Kind":"Namespace"} "roles"=null "uid"="1a91a494-b510-4e0c-b2af-5cd337c54bf4" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}
I0929 16:39:14.859021   39172 handler.go:167] webhooks/resource/validate "msg"="creating the UR to generate downstream on trigger's operation" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "gvk"={"group":"","version":"v1","kind":"Namespace"} "gvr"={"group":"","version":"v1","resource":"namespaces"} "kind"="Namespace" "name"="trainer" "namespace"="trainer" "operation"="CREATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"Namespace"} "roles"=null "rule"="cpol-data-sync-modify-downstream-rule" "uid"="1a91a494-b510-4e0c-b2af-5cd337c54bf4" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}
I0929 16:39:14.859253   39172 generator.go:42] updaterequest-generator "msg"="apply Update Request" "request"={"requestType":"generate","policy":"cpol-data-sync-modify-downstream-policy","rule":"cpol-data-sync-modify-downstream-rule","deleteDownstream":false,"resource":{"apiVersion":"v1","kind":"Namespace","name":"trainer"},"context":{"userInfo":{"clusterRoles":["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"],"userInfo":{"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}},"admissionRequestInfo":{"admissionRequest":{"uid":"1a91a494-b510-4e0c-b2af-5cd337c54bf4","kind":{"group":"","version":"v1","kind":"Namespace"},"resource":{"group":"","version":"v1","resource":"namespaces"},"requestKind":{"group":"","version":"v1","kind":"Namespace"},"requestResource":{"group":"","version":"v1","resource":"namespaces"},"name":"trainer","namespace":"trainer","operation":"CREATE","userInfo":{"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]},"object":{"kind":"Namespace","apiVersion":"v1","metadata":{"name":"trainer","uid":"093586a0-db4e-43ed-ab5b-1ba0753124e2","creationTimestamp":"2023-09-29T16:39:11Z","labels":{"kubernetes.io/metadata.name":"trainer"},"annotations":{"kubectl.kubernetes.io/last-applied-configuration":"{\"apiVersion\":\"v1\",\"kind\":\"Namespace\",\"metadata\":{\"annotations\":{},\"name\":\"trainer\"}}\n"},"managedFields":[{"manager":"kubectl-client-side-apply","operation":"Update","apiVersion":"v1","time":"2023-09-29T16:39:11Z","fieldsType":"FieldsV1","fieldsV1":{"f:metadata":{"f:annotations":{".":{},"f:kubectl.kubernetes.io/last-applied-configuration":{}},"f:labels":{".":{},"f:kubernetes.io/metadata.name":{}}}}}]},"spec":{"finalizers":["kubernetes"]},"status":{"phase":"Active"}},"oldObject":null,"dryRun":false,"options":{"kind":"CreateOptions","apiVersion":"meta.k8s.io/v1","fieldManager":"kubectl-client-side-apply","fieldValidation":"Strict"}},"operation":"CREATE"}}}
I0929 16:39:14.859324   39172 generator.go:72] updaterequest-generator "msg"="creating new UpdateRequest" "resource"="v1/Namespace//trainer" "ruleType"="generate"
I0929 16:39:14.859969   39172 handlers.go:101] webhooks/resource/validate "msg"="received an admission request in validating webhook" "clusterroles"=["system:basic-user","system:controller:root-ca-cert-publisher","system:discovery","system:public-info-viewer","system:service-account-issuer-discovery"] "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "name"="kube-root-ca.crt" "namespace"="trainer" "operation"="CREATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="cf59c0cf-0b5b-4eee-a376-5e656eb1edfa" "user"={"username":"system:serviceaccount:kube-system:root-ca-cert-publisher","uid":"ab43aa9f-672a-4918-914f-4408ae727d47","groups":["system:serviceaccounts","system:serviceaccounts:kube-system","system:authenticated"]}
I0929 16:39:14.860085   39172 handlers.go:112] webhooks/resource/validate "msg"="no policies matched admission request" "clusterroles"=["system:basic-user","system:controller:root-ca-cert-publisher","system:discovery","system:public-info-viewer","system:service-account-issuer-discovery"] "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "name"="kube-root-ca.crt" "namespace"="trainer" "operation"="CREATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="cf59c0cf-0b5b-4eee-a376-5e656eb1edfa" "user"={"username":"system:serviceaccount:kube-system:root-ca-cert-publisher","uid":"ab43aa9f-672a-4918-914f-4408ae727d47","groups":["system:serviceaccounts","system:serviceaccounts:kube-system","system:authenticated"]}
I0929 16:39:14.860260   39172 handlers.go:115] webhooks/resource/validate "msg"="processing policies for validate admission request" "clusterroles"=["system:basic-user","system:controller:root-ca-cert-publisher","system:discovery","system:public-info-viewer","system:service-account-issuer-discovery"] "generate"=0 "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "mutate"=0 "name"="kube-root-ca.crt" "namespace"="trainer" "operation"="CREATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="cf59c0cf-0b5b-4eee-a376-5e656eb1edfa" "user"={"username":"system:serviceaccount:kube-system:root-ca-cert-publisher","uid":"ab43aa9f-672a-4918-914f-4408ae727d47","groups":["system:serviceaccounts","system:serviceaccounts:kube-system","system:authenticated"]} "validate"=0
I0929 16:39:14.861414   39172 context.go:255] context "msg"="Adding service account" "service account name"="" "service account namespace"=""
I0929 16:39:14.861555   39172 controller.go:121] EventGenerator "msg"="generating events" "count"=0
I0929 16:39:14.864250   39172 context.go:255] context "msg"="Adding service account" "service account name"="root-ca-cert-publisher" "service account namespace"="kube-system"
I0929 16:39:14.864576   39172 block.go:33] webhooks/resource/validate "msg"="allowing admission request" "action"="validate" "clusterroles"=["system:basic-user","system:controller:root-ca-cert-publisher","system:discovery","system:public-info-viewer","system:service-account-issuer-discovery"] "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "name"="kube-root-ca.crt" "namespace"="trainer" "operation"="CREATE" "resource"="trainer/ConfigMap/kube-root-ca.crt" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="cf59c0cf-0b5b-4eee-a376-5e656eb1edfa" "user"={"username":"system:serviceaccount:kube-system:root-ca-cert-publisher","uid":"ab43aa9f-672a-4918-914f-4408ae727d47","groups":["system:serviceaccounts","system:serviceaccounts:kube-system","system:authenticated"]}
I0929 16:39:14.864728   39172 controller.go:121] EventGenerator "msg"="generating events" "count"=0
I0929 16:39:14.874618   39172 context.go:255] context "msg"="Adding service account" "service account name"="root-ca-cert-publisher" "service account namespace"="kube-system"
I0929 16:39:14.874750   39172 controller.go:121] EventGenerator "msg"="generating events" "count"=0
I0929 16:39:14.884866   39172 generator.go:92] updaterequest-generator "msg"="successfully created UpdateRequest" "name"="ur-m9c2s" "namespace"="kyverno" "resource"="v1/Namespace//trainer" "ruleType"="generate"
I0929 16:39:15.902857   39172 handlers.go:101] webhooks/resource/validate "msg"="received an admission request in validating webhook" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "name"="zk-kafka-address" "namespace"="trainer" "operation"="CREATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="6536bfb0-9de6-4f11-8b9d-9e2e7c3b5528" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}
I0929 16:39:15.902932   39172 handlers.go:112] webhooks/resource/validate "msg"="no policies matched admission request" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "name"="zk-kafka-address" "namespace"="trainer" "operation"="CREATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="6536bfb0-9de6-4f11-8b9d-9e2e7c3b5528" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}
I0929 16:39:15.903040   39172 handlers.go:115] webhooks/resource/validate "msg"="processing policies for validate admission request" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "generate"=0 "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "mutate"=0 "name"="zk-kafka-address" "namespace"="trainer" "operation"="CREATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="6536bfb0-9de6-4f11-8b9d-9e2e7c3b5528" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]} "validate"=0
I0929 16:39:15.905621   39172 context.go:255] context "msg"="Adding service account" "service account name"="" "service account namespace"=""
I0929 16:39:15.905799   39172 block.go:33] webhooks/resource/validate "msg"="allowing admission request" "action"="validate" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "name"="zk-kafka-address" "namespace"="trainer" "operation"="CREATE" "resource"="trainer/ConfigMap/zk-kafka-address" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="6536bfb0-9de6-4f11-8b9d-9e2e7c3b5528" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}
I0929 16:39:15.905840   39172 controller.go:121] EventGenerator "msg"="generating events" "count"=0
I0929 16:39:15.908953   39172 context.go:255] context "msg"="Adding service account" "service account name"="" "service account namespace"=""
I0929 16:39:15.909082   39172 controller.go:121] EventGenerator "msg"="generating events" "count"=0
I0929 16:40:08.080303   39172 handlers.go:101] webhooks/resource/validate "msg"="received an admission request in validating webhook" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "name"="zk-kafka-address" "namespace"="trainer" "operation"="UPDATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="00fc452c-0c6c-443b-a92d-625fa591c688" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}
I0929 16:40:08.080388   39172 handlers.go:112] webhooks/resource/validate "msg"="no policies matched admission request" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "name"="zk-kafka-address" "namespace"="trainer" "operation"="UPDATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="00fc452c-0c6c-443b-a92d-625fa591c688" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}
I0929 16:40:08.080470   39172 handlers.go:115] webhooks/resource/validate "msg"="processing policies for validate admission request" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "generate"=0 "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "mutate"=0 "name"="zk-kafka-address" "namespace"="trainer" "operation"="UPDATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="00fc452c-0c6c-443b-a92d-625fa591c688" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]} "validate"=0
I0929 16:40:08.085089   39172 context.go:255] context "msg"="Adding service account" "service account name"="" "service account namespace"=""
I0929 16:40:08.085385   39172 block.go:33] webhooks/resource/validate "msg"="allowing admission request" "action"="validate" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "name"="zk-kafka-address" "namespace"="trainer" "operation"="UPDATE" "resource"="trainer/ConfigMap/zk-kafka-address" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="00fc452c-0c6c-443b-a92d-625fa591c688" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}
I0929 16:40:08.085419   39172 controller.go:121] EventGenerator "msg"="generating events" "count"=0
I0929 16:40:08.085791   39172 handler.go:138] webhooks/resource/validate "msg"="handle non-trigger resource operation for generate" "clusterroles"=["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"] "gvk"={"group":"","version":"v1","kind":"ConfigMap"} "gvr"={"group":"","version":"v1","resource":"configmaps"} "kind"="ConfigMap" "name"="zk-kafka-address" "namespace"="trainer" "operation"="UPDATE" "resource.gvk"={"Group":"","Version":"v1","Kind":"ConfigMap"} "roles"=null "uid"="00fc452c-0c6c-443b-a92d-625fa591c688" "user"={"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}
I0929 16:40:08.086014   39172 generator.go:42] updaterequest-generator "msg"="apply Update Request" "request"={"requestType":"generate","policy":"cpol-data-sync-modify-downstream-policy","rule":"cpol-data-sync-modify-downstream-rule","deleteDownstream":false,"resource":{"apiVersion":"v1","kind":"Namespace","name":"trainer"},"context":{"userInfo":{"clusterRoles":["cluster-admin","system:basic-user","system:discovery","system:public-info-viewer"],"userInfo":{"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]}},"admissionRequestInfo":{"admissionRequest":{"uid":"00fc452c-0c6c-443b-a92d-625fa591c688","kind":{"group":"","version":"v1","kind":"ConfigMap"},"resource":{"group":"","version":"v1","resource":"configmaps"},"requestKind":{"group":"","version":"v1","kind":"ConfigMap"},"requestResource":{"group":"","version":"v1","resource":"configmaps"},"name":"zk-kafka-address","namespace":"trainer","operation":"UPDATE","userInfo":{"username":"kubernetes-admin","groups":["system:masters","system:authenticated"]},"object":{"kind":"ConfigMap","apiVersion":"v1","metadata":{"name":"zk-kafka-address","namespace":"trainer","uid":"ab6e86c0-35ba-4fe5-96c5-739e800fbcca","resourceVersion":"1177","creationTimestamp":"2023-09-29T16:39:15Z","labels":{"app.kubernetes.io/managed-by":"kyverno","generate.kyverno.io/policy-name":"cpol-data-sync-modify-downstream-policy","generate.kyverno.io/policy-namespace":"","generate.kyverno.io/rule-name":"cpol-data-sync-modify-downstream-rule","generate.kyverno.io/trigger-group":"","generate.kyverno.io/trigger-kind":"Namespace","generate.kyverno.io/trigger-name":"trainer","generate.kyverno.io/trigger-namespace":"","generate.kyverno.io/trigger-version":"v1","somekey":"somevalue"},"annotations":{"kubectl.kubernetes.io/last-applied-configuration":"{\"apiVersion\":\"v1\",\"data\":{\"KAFKA_ADDRESS\":\"192.168.10.13:9092,192.168.10.14:9092,192.168.10.15:9092\",\"ZK_ADDRESS\":\"ichangedthis\"},\"kind\":\"ConfigMap\",\"metadata\":{\"annotations\":{},\"labels\":{\"somekey\":\"somevalue\"},\"name\":\"zk-kafka-address\",\"namespace\":\"trainer\"}}\n"},"managedFields":[{"manager":"__debug_bin4271173230","operation":"Update","apiVersion":"v1","time":"2023-09-29T16:39:15Z","fieldsType":"FieldsV1","fieldsV1":{"f:data":{".":{},"f:KAFKA_ADDRESS":{}},"f:metadata":{"f:labels":{".":{},"f:app.kubernetes.io/managed-by":{},"f:generate.kyverno.io/policy-name":{},"f:generate.kyverno.io/policy-namespace":{},"f:generate.kyverno.io/rule-name":{},"f:generate.kyverno.io/trigger-group":{},"f:generate.kyverno.io/trigger-kind":{},"f:generate.kyverno.io/trigger-name":{},"f:generate.kyverno.io/trigger-namespace":{},"f:generate.kyverno.io/trigger-version":{},"f:somekey":{}}}}},{"manager":"kubectl-client-side-apply","operation":"Update","apiVersion":"v1","time":"2023-09-29T16:40:08Z","fieldsType":"FieldsV1","fieldsV1":{"f:data":{"f:ZK_ADDRESS":{}},"f:metadata":{"f:annotations":{".":{},"f:kubectl.kubernetes.io/last-applied-configuration":{}}}}}]},"data":{"KAFKA_ADDRESS":"192.168.10.13:9092,192.168.10.14:9092,192.168.10.15:9092","ZK_ADDRESS":"ichangedthis"}},"oldObject":{"kind":"ConfigMap","apiVersion":"v1","metadata":{"name":"zk-kafka-address","namespace":"trainer","uid":"ab6e86c0-35ba-4fe5-96c5-739e800fbcca","resourceVersion":"1177","creationTimestamp":"2023-09-29T16:39:15Z","labels":{"app.kubernetes.io/managed-by":"kyverno","generate.kyverno.io/policy-name":"cpol-data-sync-modify-downstream-policy","generate.kyverno.io/policy-namespace":"","generate.kyverno.io/rule-name":"cpol-data-sync-modify-downstream-rule","generate.kyverno.io/trigger-group":"","generate.kyverno.io/trigger-kind":"Namespace","generate.kyverno.io/trigger-name":"trainer","generate.kyverno.io/trigger-namespace":"","generate.kyverno.io/trigger-version":"v1","somekey":"somevalue"},"managedFields":[{"manager":"__debug_bin4271173230","operation":"Update","apiVersion":"v1","time":"2023-09-29T16:39:15Z","fieldsType":"FieldsV1","fieldsV1":{"f:data":{".":{},"f:KAFKA_ADDRESS":{},"f:ZK_ADDRESS":{}},"f:metadata":{"f:labels":{".":{},"f:app.kubernetes.io/managed-by":{},"f:generate.kyverno.io/policy-name":{},"f:generate.kyverno.io/policy-namespace":{},"f:generate.kyverno.io/rule-name":{},"f:generate.kyverno.io/trigger-group":{},"f:generate.kyverno.io/trigger-kind":{},"f:generate.kyverno.io/trigger-name":{},"f:generate.kyverno.io/trigger-namespace":{},"f:generate.kyverno.io/trigger-version":{},"f:somekey":{}}}}}]},"data":{"KAFKA_ADDRESS":"192.168.10.13:9092,192.168.10.14:9092,192.168.10.15:9092","ZK_ADDRESS":"192.168.10.10:2181,192.168.10.11:2181,192.168.10.12:2181"}},"dryRun":false,"options":{"kind":"UpdateOptions","apiVersion":"meta.k8s.io/v1","fieldManager":"kubectl-client-side-apply","fieldValidation":"Strict"}},"operation":"UPDATE"}}}
I0929 16:40:08.086111   39172 generator.go:72] updaterequest-generator "msg"="creating new UpdateRequest" "resource"="v1/Namespace//trainer" "ruleType"="generate"
I0929 16:40:08.091631   39172 context.go:255] context "msg"="Adding service account" "service account name"="" "service account namespace"=""
I0929 16:40:08.091896   39172 controller.go:121] EventGenerator "msg"="generating events" "count"=0
I0929 16:40:08.108002   39172 generator.go:92] updaterequest-generator "msg"="successfully created UpdateRequest" "name"="ur-pffn9" "namespace"="kyverno" "resource"="v1/Namespace//trainer" "ruleType"="generate"
```

Here's the logs from the Kyverno Background Controller in `level 4`:

```
I0929 16:37:23.482864   39170 version.go:49] setup/version "msg"="version" "hash"="c023f972ba1708fca2896ef4f14bbad04fe3cb07" "time"="2023-09-29T10:52:47Z" "version"="(devel)"
I0929 16:37:23.482924   39170 flag.go:240] setup/flag "msg"="" "add_dir_header"=false
I0929 16:37:23.482945   39170 flag.go:240] setup/flag "msg"="" "allowInsecureRegistry"=false
I0929 16:37:23.482964   39170 flag.go:240] setup/flag "msg"="" "alsologtostderr"=false
I0929 16:37:23.482990   39170 flag.go:240] setup/flag "msg"="" "clientRateLimitBurst"=50
I0929 16:37:23.483016   39170 flag.go:240] setup/flag "msg"="" "clientRateLimitQPS"=20
I0929 16:37:23.483034   39170 flag.go:240] setup/flag "msg"="" "disableMetrics"=false
I0929 16:37:23.483051   39170 flag.go:240] setup/flag "msg"="" "enableConfigMapCaching"=true
I0929 16:37:23.483088   39170 flag.go:240] setup/flag "msg"="" "enableDeferredLoading"=
I0929 16:37:23.483112   39170 flag.go:240] setup/flag "msg"="" "enablePolicyException"=true
I0929 16:37:23.483142   39170 flag.go:240] setup/flag "msg"="" "enableTracing"=false
I0929 16:37:23.483178   39170 flag.go:240] setup/flag "msg"="" "exceptionNamespace"=""
I0929 16:37:23.483203   39170 flag.go:240] setup/flag "msg"="" "genWorkers"=10
I0929 16:37:23.483228   39170 flag.go:240] setup/flag "msg"="" "imagePullSecrets"=""
I0929 16:37:23.483257   39170 flag.go:240] setup/flag "msg"="" "kubeconfig"="/home/ubuntu/.kube/config"
I0929 16:37:23.483297   39170 flag.go:240] setup/flag "msg"="" "leaderElectionRetryPeriod"=2000000000
I0929 16:37:23.483345   39170 flag.go:240] setup/flag "msg"="" "log_backtrace_at"={}
I0929 16:37:23.483378   39170 flag.go:240] setup/flag "msg"="" "log_dir"=""
I0929 16:37:23.483406   39170 flag.go:240] setup/flag "msg"="" "log_file"=""
I0929 16:37:23.483448   39170 flag.go:240] setup/flag "msg"="" "log_file_max_size"=1800
I0929 16:37:23.483484   39170 flag.go:240] setup/flag "msg"="" "loggingFormat"="text"
I0929 16:37:23.483516   39170 flag.go:240] setup/flag "msg"="" "logtostderr"=true
I0929 16:37:23.483548   39170 flag.go:240] setup/flag "msg"="" "maxQueuedEvents"=1000
I0929 16:37:23.483580   39170 flag.go:240] setup/flag "msg"="" "metricsPort"="8000"
I0929 16:37:23.483612   39170 flag.go:240] setup/flag "msg"="" "omit-events"=""
I0929 16:37:23.483644   39170 flag.go:240] setup/flag "msg"="" "one_output"=false
I0929 16:37:23.483667   39170 flag.go:240] setup/flag "msg"="" "otelCollector"="opentelemetrycollector.kyverno.svc.cluster.local"
I0929 16:37:23.483700   39170 flag.go:240] setup/flag "msg"="" "otelConfig"="prometheus"
I0929 16:37:23.483734   39170 flag.go:240] setup/flag "msg"="" "profile"=false
I0929 16:37:23.483767   39170 flag.go:240] setup/flag "msg"="" "profileAddress"=""
I0929 16:37:23.483807   39170 flag.go:240] setup/flag "msg"="" "profilePort"="6060"
I0929 16:37:23.483839   39170 flag.go:240] setup/flag "msg"="" "registryCredentialHelpers"=""
I0929 16:37:23.483865   39170 flag.go:240] setup/flag "msg"="" "skip_headers"=false
I0929 16:37:23.483889   39170 flag.go:240] setup/flag "msg"="" "skip_log_headers"=false
I0929 16:37:23.483939   39170 flag.go:240] setup/flag "msg"="" "stderrthreshold"={"Severity":2}
I0929 16:37:23.483975   39170 flag.go:240] setup/flag "msg"="" "tracingAddress"=""
I0929 16:37:23.484023   39170 flag.go:240] setup/flag "msg"="" "tracingCreds"=""
I0929 16:37:23.484053   39170 flag.go:240] setup/flag "msg"="" "tracingPort"="4317"
I0929 16:37:23.484086   39170 flag.go:240] setup/flag "msg"="" "transportCreds"=""
I0929 16:37:23.484127   39170 flag.go:240] setup/flag "msg"="" "v"=4
I0929 16:37:23.484176   39170 flag.go:240] setup/flag "msg"="" "vmodule"={}
I0929 16:37:23.484215   39170 maxprocs.go:12] setup/maxprocs "msg"="setup maxprocs..." 
I0929 16:37:23.484590   39170 maxprocs.go:16] setup/maxprocs "msg"="maxprocs: Leaving GOMAXPROCS=4: CPU quota undefined" 
I0929 16:37:23.484628   39170 signal.go:16] setup/signals "msg"="setup signals..." 
I0929 16:37:23.485107   39170 client.go:41] setup/kube-client "msg"="create kube client..." "burst"=50 "kubeconfig"="/home/ubuntu/.kube/config" "qps"=20
I0929 16:37:23.488447   39170 reflector.go:289] Starting reflector *v1.ConfigMap (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.488484   39170 reflector.go:325] Listing and watching *v1.ConfigMap from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.589203   39170 shared_informer.go:341] caches populated
I0929 16:37:23.589293   39170 metricsconfig.go:116] config "msg"="metricsRefreshInterval not set" "name"="kyverno-metrics" "namespace"="kyverno"
I0929 16:37:23.589436   39170 metricsconfig.go:138] config "msg"="namespaces configured" "name"="kyverno-metrics" "namespace"="kyverno" "namespaces"="{\"exclude\":[],\"include\":[]}"
I0929 16:37:23.589507   39170 metrics.go:18] setup/metrics "msg"="setup metrics..." "collector"="opentelemetrycollector.kyverno.svc.cluster.local" "creds"="" "otel"="prometheus" "port"="8000"
I0929 16:37:23.589602   39170 run.go:58] metrics-config-controller "msg"="starting ..." 
I0929 16:37:23.589781   39170 run.go:71] metrics-config-controller/worker "msg"="starting worker" "id"=0
I0929 16:37:23.590080   39170 reflector.go:289] Starting reflector *v1.ConfigMap (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.590129   39170 reflector.go:325] Listing and watching *v1.ConfigMap from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.690098   39170 shared_informer.go:341] caches populated
I0929 16:37:23.691166   39170 config.go:338] config "msg"="filters configured" "filters"=[{"Group":"*","Version":"*","Kind":"*","Subresource":"*","Namespace":"kyverno","Name":"*"},{"Group":"*","Version":"*","Kind":"Event","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"*","Subresource":"*","Namespace":"kube-system","Name":"*"},{"Group":"*","Version":"*","Kind":"*","Subresource":"*","Namespace":"kube-public","Name":"*"},{"Group":"*","Version":"*","Kind":"*","Subresource":"*","Namespace":"kube-node-lease","Name":"*"},{"Group":"*","Version":"*","Kind":"Node","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"Node","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"APIService","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"APIService","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"TokenReview","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"SubjectAccessReview","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"SelfSubjectAccessReview","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"Binding","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"binding","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ReplicaSet","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ReplicaSet","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"AdmissionReport","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"AdmissionReport","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterAdmissionReport","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterAdmissionReport","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"BackgroundScanReport","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"BackgroundScanReport","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterBackgroundScanReport","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterBackgroundScanReport","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:admission-controller"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:admission-controller:core"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:admission-controller:additional"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:background-controller"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:background-controller:core"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:background-controller:additional"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:cleanup-controller"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:cleanup-controller:core"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:cleanup-controller:additional"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:reports-controller"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:reports-controller:core"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:reports-controller:additional"},{"Group":"*","Version":"*","Kind":"ClusterRoleBinding","Subresource":"","Namespace":"*","Name":"kyverno:admission-controller"},{"Group":"*","Version":"*","Kind":"ClusterRoleBinding","Subresource":"","Namespace":"*","Name":"kyverno:background-controller"},{"Group":"*","Version":"*","Kind":"ClusterRoleBinding","Subresource":"","Namespace":"*","Name":"kyverno:cleanup-controller"},{"Group":"*","Version":"*","Kind":"ClusterRoleBinding","Subresource":"","Namespace":"*","Name":"kyverno:reports-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Role","Subresource":"","Namespace":"kyverno","Name":"kyverno:admission-controller"},{"Group":"*","Version":"*","Kind":"Role","Subresource":"","Namespace":"kyverno","Name":"kyverno:background-controller"},{"Group":"*","Version":"*","Kind":"Role","Subresource":"","Namespace":"kyverno","Name":"kyverno:cleanup-controller"},{"Group":"*","Version":"*","Kind":"Role","Subresource":"","Namespace":"kyverno","Name":"kyverno:reports-controller"},{"Group":"*","Version":"*","Kind":"RoleBinding","Subresource":"","Namespace":"kyverno","Name":"kyverno:admission-controller"},{"Group":"*","Version":"*","Kind":"RoleBinding","Subresource":"","Namespace":"kyverno","Name":"kyverno:background-controller"},{"Group":"*","Version":"*","Kind":"RoleBinding","Subresource":"","Namespace":"kyverno","Name":"kyverno:cleanup-controller"},{"Group":"*","Version":"*","Kind":"RoleBinding","Subresource":"","Namespace":"kyverno","Name":"kyverno:reports-controller"},{"Group":"*","Version":"*","Kind":"ConfigMap","Subresource":"","Namespace":"kyverno","Name":"kyverno"},{"Group":"*","Version":"*","Kind":"ConfigMap","Subresource":"","Namespace":"kyverno","Name":"kyverno-metrics"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller-*"},{"Group":"*","Version":"*","Kind":"Job","Subresource":"","Namespace":"kyverno","Name":"kyverno-hook-pre-delete"},{"Group":"*","Version":"*","Kind":"Job","Subresource":"*","Namespace":"kyverno","Name":"kyverno-hook-pre-delete"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-svc"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-svc"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-svc-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-svc-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller-metrics"},{"Group":"*","Version":"*","Kind":"ServiceMonitor","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"ServiceMonitor","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"ServiceMonitor","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"ServiceMonitor","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Secret","Subresource":"","Namespace":"kyverno","Name":"kyverno-svc.kyverno.svc.*"},{"Group":"*","Version":"*","Kind":"Secret","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller.kyverno.svc.*"}] "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.691334   39170 config.go:347] config "msg"="defaultRegistry configured" "defaultRegistry"="docker.io" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.691378   39170 config.go:363] config "msg"="enableDefaultRegistryMutation configured" "enableDefaultRegistryMutation"="true" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.691422   39170 config.go:372] config "msg"="excludedGroups configured" "excludeGroups"=["system:nodes"] "includeGroups"=null "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.691455   39170 config.go:377] config "msg"="excludeUsernames not set" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.691496   39170 config.go:385] config "msg"="excludeRoles not set" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.691530   39170 config.go:393] config "msg"="excludeClusterRoles not set" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.691569   39170 config.go:409] config "msg"="generateSuccessEvents configured" "generateSuccessEvents"="false" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.691741   39170 config.go:423] config "msg"="webhooks configured" "name"="kyverno" "namespace"="kyverno" "webhooks"="[{\"namespaceSelector\": {\"matchExpressions\": [{\"key\":\"kubernetes.io/metadata.name\",\"operator\":\"NotIn\",\"values\":[\"kyverno\"]}]}}]"
I0929 16:37:23.691787   39170 config.go:429] config "msg"="webhookAnnotations not set" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.691821   39170 config.go:443] config "msg"="matchConditions not set" "name"="kyverno" "namespace"="kyverno"
I0929 16:37:23.691881   39170 registry.go:18] setup/registry-client "msg"="setup registry client..." "insecure"=false "secrets"=""
I0929 16:37:23.692017   39170 run.go:58] config-controller "msg"="starting ..." 
I0929 16:37:23.692078   39170 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:23.692075   39170 reflector.go:289] Starting reflector *v1.Secret (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.692121   39170 reflector.go:325] Listing and watching *v1.Secret from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.692184   39170 run.go:71] config-controller/worker "msg"="starting worker" "id"=0
I0929 16:37:23.792319   39170 shared_informer.go:341] caches populated
I0929 16:37:23.792396   39170 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:23.792447   39170 client.go:41] setup/kube-client "msg"="create kube client..." "burst"=50 "kubeconfig"="/home/ubuntu/.kube/config" "qps"=20
I0929 16:37:23.793859   39170 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:23.794187   39170 client.go:49] setup/kyverno-client "msg"="create kyverno client..." "burst"=50 "kubeconfig"="/home/ubuntu/.kube/config" "qps"=20
I0929 16:37:23.795411   39170 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:23.795531   39170 client.go:57] setup/dynamic-client "msg"="create dynamic client..." "burst"=50 "kubeconfig"="/home/ubuntu/.kube/config" "qps"=20
I0929 16:37:23.796713   39170 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp" "SchemaURL"="" "Version"="0.44.0"
I0929 16:37:23.796768   39170 client.go:81] setup/d-client "msg"="create the kyverno dynamic client..." "burst"=50 "kubeconfig"="/home/ubuntu/.kube/config" "qps"=20
I0929 16:37:23.796838   39170 main.go:125] setup "msg"="setting the background scan interval" "value"="1h0m0s"
I0929 16:37:23.977529   39170 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:23.977632   39170 engine.go:87] setup/configmap-resolver "msg"="setup config map resolver..." "enableConfigMapCaching"=true
I0929 16:37:23.977836   39170 reflector.go:289] Starting reflector *v1.ConfigMap (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:23.977875   39170 reflector.go:325] Listing and watching *v1.ConfigMap from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.077914   39170 shared_informer.go:341] caches populated
I0929 16:37:24.078004   39170 engine.go:62] setup/exception-selector "msg"="setup exception selector..." "enablePolicyException"=true "exceptionNamespace"=""
I0929 16:37:24.078145   39170 reflector.go:289] Starting reflector *v2alpha1.PolicyException (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.078188   39170 reflector.go:325] Listing and watching *v2alpha1.PolicyException from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.178621   39170 shared_informer.go:341] caches populated
I0929 16:37:24.178702   39170 engine.go:41] setup/engine "msg"="setup engine..." 
I0929 16:37:24.178748   39170 internal_logging.go:52] setup/metrics "msg"="Meter created" "Name"="kyverno" "SchemaURL"="" "Version"=""
I0929 16:37:24.178878   39170 reflector.go:289] Starting reflector *v1.ClusterPolicy (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.178920   39170 reflector.go:325] Listing and watching *v1.ClusterPolicy from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.178995   39170 reflector.go:289] Starting reflector *v1.Policy (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.179028   39170 reflector.go:325] Listing and watching *v1.Policy from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.279638   39170 shared_informer.go:341] caches populated
I0929 16:37:24.380180   39170 shared_informer.go:341] caches populated
I0929 16:37:24.380256   39170 leaderelection.go:250] attempting to acquire leader lease kyverno/kyverno-background-controller...
I0929 16:37:24.380623   39170 controller.go:152] EventGenerator "msg"="start" 
I0929 16:37:24.387632   39170 leaderelection.go:260] successfully acquired lease kyverno/kyverno-background-controller
I0929 16:37:24.387909   39170 leaderelection.go:97] setup/leader-election "msg"="still leading" "id"="dummy"
I0929 16:37:24.387945   39170 leaderelection.go:83] setup/leader-election "msg"="started leading" "id"="dummy"
I0929 16:37:24.388202   39170 reflector.go:289] Starting reflector *v1.ClusterPolicy (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.388234   39170 reflector.go:325] Listing and watching *v1.ClusterPolicy from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.388453   39170 reflector.go:289] Starting reflector *v1.Policy (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.388473   39170 reflector.go:325] Listing and watching *v1.Policy from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.388695   39170 reflector.go:289] Starting reflector *v1beta1.UpdateRequest (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.388714   39170 reflector.go:325] Listing and watching *v1beta1.UpdateRequest from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.388921   39170 reflector.go:289] Starting reflector *v1.Namespace (15m0s) from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.388956   39170 reflector.go:325] Listing and watching *v1.Namespace from pkg/mod/k8s.io/client-go@v0.28.2/tools/cache/reflector.go:229
I0929 16:37:24.488757   39170 shared_informer.go:341] caches populated
I0929 16:37:24.488961   39170 shared_informer.go:341] caches populated
I0929 16:37:24.488980   39170 shared_informer.go:341] caches populated
I0929 16:37:24.489002   39170 shared_informer.go:341] caches populated
I0929 16:37:24.489116   39170 controller.go:32] setup/leader/controllers "msg"="starting controller" "name"="background-controller" "workers"=10
I0929 16:37:24.489162   39170 update_request_controller.go:106] background "msg"="starting" 
I0929 16:37:24.489185   39170 shared_informer.go:311] Waiting for caches to sync for background
I0929 16:37:24.489207   39170 shared_informer.go:341] caches populated
I0929 16:37:24.489215   39170 shared_informer.go:318] Caches are synced for background
I0929 16:37:24.489116   39170 controller.go:32] setup/leader/controllers "msg"="starting controller" "name"="policy-controller" "workers"=2
I0929 16:37:24.489270   39170 policy_controller.go:236] PolicyController "msg"="starting" 
I0929 16:37:24.489315   39170 shared_informer.go:311] Waiting for caches to sync for PolicyController
I0929 16:37:24.489366   39170 shared_informer.go:341] caches populated
I0929 16:37:24.489387   39170 shared_informer.go:318] Caches are synced for PolicyController
I0929 16:38:40.089240   39170 policy_controller.go:165] PolicyController "msg"="policy created" "kind"="ClusterPolicy" "name"="cpol-data-sync-modify-downstream-policy" "namespace"="" "uid"="c00d2520-5f8e-4725-8665-19f7e3e4147c"
I0929 16:38:40.092233   39170 vars.go:375]  "msg"="variable substituted" "path"="/generate/namespace" "value"=null "variable"="{{request.object.metadata.name}}"
I0929 16:38:40.092609   39170 policy_controller.go:171] PolicyController "msg"="queuing policy for background processing" "name"="cpol-data-sync-modify-downstream-policy"
I0929 16:38:40.092746   39170 policy_controller.go:304] PolicyController/syncPolicy "msg"="started syncing policy" "key"="cpol-data-sync-modify-downstream-policy" "startTime"="2023-09-29T16:38:40.092667429Z"
I0929 16:38:40.092798   39170 mutate.go:16] PolicyController/handleMutate/cpol-data-sync-modify-downstream-policy "msg"="skip policy application on policy event" "mutateExiting"=false "policyKey"="cpol-data-sync-modify-downstream-policy"
I0929 16:38:40.092826   39170 generate.go:20] PolicyController/handleGenerate/cpol-data-sync-modify-downstream-policy "msg"="update URs on policy event" 
I0929 16:38:40.104226   39170 vars.go:375]  "msg"="variable substituted" "path"="/generate/namespace" "value"=null "variable"="{{request.object.metadata.name}}"
I0929 16:38:40.116183   39170 generate.go:109] PolicyController "msg"="sync data rule changes to downstream targets" 
I0929 16:38:40.116238   39170 policy_controller.go:306] PolicyController/syncPolicy "msg"="finished syncing policy" "key"="cpol-data-sync-modify-downstream-policy" "processingTime"="23.54596ms"
I0929 16:39:14.877229   39170 update_request_controller.go:163] background "msg"="started sync" "key"="kyverno/ur-m9c2s" "startTime"="2023-09-29T16:39:14.877169319Z"
I0929 16:39:14.882100   39170 update_request_controller.go:192] background "msg"="synced update request" "key"="kyverno/ur-m9c2s" "processingTime"="4.895818ms" "ur status"=""
I0929 16:39:14.885512   39170 update_request_controller.go:163] background "msg"="started sync" "key"="kyverno/ur-m9c2s" "startTime"="2023-09-29T16:39:14.885463639Z"
I0929 16:39:14.885597   39170 generate.go:97] background "msg"="start processing UR" "name"="ur-m9c2s" "policy"="cpol-data-sync-modify-downstream-policy" "resource"="v1/Namespace//trainer" "resourceVersion"="1174" "ur"="ur-m9c2s"
I0929 16:39:15.889840   39170 resource.go:60] background "msg"="fetched trigger resource" "resourceSpec"={"apiVersion":"v1","kind":"Namespace","name":"trainer"}
I0929 16:39:15.889899   39170 generate.go:193] background "msg"="applying generate policy rule" "name"="ur-m9c2s" "policy"="cpol-data-sync-modify-downstream-policy" "resource"="v1/Namespace//trainer"
I0929 16:39:15.892592   39170 context.go:255] context "msg"="Adding service account" "service account name"="" "service account namespace"=""
I0929 16:39:15.896698   39170 vars.go:375] background "msg"="variable substituted" "name"="ur-m9c2s" "path"="/generate/namespace" "policy"="cpol-data-sync-modify-downstream-policy" "resource"="v1/Namespace//trainer" "value"="trainer" "variable"="{{request.object.metadata.name}}"
I0929 16:39:15.899915   39170 generate.go:388] background "msg"="applying generate rule" "mode"="CREATE" "name"="ur-m9c2s" "policy"="cpol-data-sync-modify-downstream-policy" "resource"="v1/Namespace//trainer" "target"="v1/ConfigMap/trainer/zk-kafka-address"
I0929 16:39:15.908512   39170 generate.go:416] background "msg"="created generate target resource" "name"="ur-m9c2s" "policy"="cpol-data-sync-modify-downstream-policy" "resource"="v1/Namespace//trainer" "target"="v1/ConfigMap/trainer/zk-kafka-address"
I0929 16:39:15.908560   39170 controller.go:121] EventGenerator "msg"="generating events" "count"=1
I0929 16:39:15.908615   39170 controller.go:121] EventGenerator "msg"="generating events" "count"=1
I0929 16:39:15.908964   39170 controller.go:259] EventGenerator "msg"="creating the event" "resource"="ClusterPolicy/cpol-data-sync-modify-downstream-policy" "source"="kyverno-generate" "type"="Normal"
I0929 16:39:15.909077   39170 event_broadcaster.go:318] "Event occurred" object="cpol-data-sync-modify-downstream-policy" kind="ClusterPolicy" apiVersion="kyverno.io/v1" type="Normal" reason="PolicyApplied" action="Resource Generated" note="resource generated"
I0929 16:39:15.912242   39170 controller.go:259] EventGenerator "msg"="creating the event" "resource"="ConfigMap/trainer/zk-kafka-address" "source"="kyverno-generate" "type"="Normal"
I0929 16:39:15.913083   39170 event_broadcaster.go:318] "Event occurred" object="trainer/zk-kafka-address" kind="ConfigMap" apiVersion="v1" type="Normal" reason="PolicyApplied" action="None" note="Created ConfigMap zk-kafka-address as a result of applying policy cpol-data-sync-modify-downstream-policy/cpol-data-sync-modify-downstream-rule"
I0929 16:39:15.921593   39170 util.go:37]  "msg"="updated update request status" "name"="ur-m9c2s" "state"="Completed" "status"="Completed"
I0929 16:39:15.929528   39170 update_request_controller.go:192] background "msg"="synced update request" "key"="kyverno/ur-m9c2s" "processingTime"="1.044025946s" "ur status"="Completed"
I0929 16:40:08.097707   39170 update_request_controller.go:163] background "msg"="started sync" "key"="kyverno/ur-pffn9" "startTime"="2023-09-29T16:40:08.097640395Z"
I0929 16:40:08.101987   39170 update_request_controller.go:192] background "msg"="synced update request" "key"="kyverno/ur-pffn9" "processingTime"="4.293262ms" "ur status"=""
I0929 16:40:08.109080   39170 update_request_controller.go:163] background "msg"="started sync" "key"="kyverno/ur-pffn9" "startTime"="2023-09-29T16:40:08.109013288Z"
I0929 16:40:08.109174   39170 generate.go:97] background "msg"="start processing UR" "name"="ur-pffn9" "policy"="cpol-data-sync-modify-downstream-policy" "resource"="v1/Namespace//trainer" "resourceVersion"="1339" "ur"="ur-pffn9"
I0929 16:40:08.109538   39170 generate.go:193] background "msg"="applying generate policy rule" "name"="ur-pffn9" "policy"="cpol-data-sync-modify-downstream-policy" "resource"="v1/Namespace//trainer"
I0929 16:40:08.114533   39170 context.go:255] context "msg"="Adding service account" "service account name"="" "service account namespace"=""
I0929 16:40:08.125112   39170 background.go:83] engine.generate "msg"="rule not matched" "new.kind"="ConfigMap" "new.name"="zk-kafka-address" "new.namespace"="trainer" "old.kind"="ConfigMap" "old.name"="zk-kafka-address" "old.namespace"="trainer" "policy.apply"="All" "policy.name"="cpol-data-sync-modify-downstream-policy" "policy.namespace"="" "reason"="rule cpol-data-sync-modify-downstream-rule not matched:\n 1. no resource matched" "rule.name"="cpol-data-sync-modify-downstream-rule"
I0929 16:40:08.125175   39170 generate.go:224] background "msg"="policy does not apply to resource" "name"="ur-pffn9" "policy"="cpol-data-sync-modify-downstream-policy" "resource"="v1/Namespace//trainer"
I0929 16:40:08.125223   39170 generate.go:116] background "msg"="skipping updating status of update request" "name"="ur-pffn9" "policy"="cpol-data-sync-modify-downstream-policy" "resource"="v1/Namespace//trainer"
```